src/transformers/models/auto/configuration_auto.py

```diff
@@ -572,6 +574,8 @@
         ("donut-swin", "donut"),
         ("maskformer-swin", "maskformer"),
         ("xclip", "x_clip"),
+        ("decision_transformer_gpt2", "decision_transformer"),
+        # ("nllb", "nllb"),  # Question: I am not sure how to deal with this.
```

Comment:
`nllb` has its own module directory (contains only tokenizer), but `decision_transformer_gpt2` is inside the module directory `decision_transformer` (as a modeling file)